### PR TITLE
Add hashtags

### LINKED ISSUE #1915: [Bug] Broken hashtag links in description
**Describe the bug**
The "hashtag" links in video description do not work

**Steps to Reproduce**
1. Go to some video with hashtags in description (e.g: `/watch?v=cVYf2OXRXjs`)
2. Click on hashtag (link points to `/hastag/<hastag>`)
3. Link redirects to `/feed/popular`

**Additional context**
If fixing this, also check #442.

### PR DESCRIPTION
**Pull request by SamantazFox**
Add support for hashtags.

~~Closes 442~~ (Doesn't fix it, the problem lies elsewhere)
Closes #1915

### CODE REVIEW

**Review by matthewmcgarvey — APPROVED** [review]
LGTM :shipit:

File: src/invidious/videos.cr [hunk 1]
```diff
@@ -868,11 +868,7 @@ def parse_related_video(related : JSON::Any) : Hash(String, JSON::Any)?
     .try &.dig?("runs", 0)
 
   author = channel_info.try &.dig?("text")
-  author_verified_badge = related["ownerBadges"]?.try do |badges_array|
-    badges_array.as_a.find(&.dig("metadataBadgeRenderer", "tooltip").as_s.== "Verified")
-  end
-
-  author_verified = (author_verified_badge && author_verified_badge.size > 0).to_s
+  author_verified = has_verified_badge?(related["ownerBadges"]?)
```
[the remark this answers is not in the document]
Comment (SamantazFox) [reply src/invidious/videos.cr:871]: Ah, I forgot to push the patch :X

**Review by matthewmcgarvey — APPROVED** [review]

File: src/invidious/videos.cr (hunk 1 above)
[the remark this answers is not in the document]
Comment (AHOHNMYC) [reply src/invidious/videos.cr]: Nice! Working again! Thanks

**Comment by SamantazFox** [comment]
Simple rebase, as there was a conflict on `ci.yml`

**Pull request merged**